migrations/20200529123451_create_user_table.ts

```diff
@@ -0,0 +1,15 @@
+import * as Knex from "knex";
+
+const userTableName = "user";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable(userTableName, (builder) => {
+    builder.increments("id").primary();
+    builder.string("email", 320).unique(); // max total length of email is 320 characters
```

Comment:
In Postgres, TEXT data type does not have a practical limit on its length, so there's no need to limit it.
Also, one account may have multiple email addresses in GitHub, so I don't think we should store it in DB. Instead, we can just omit this column.